src/components/clusters/wizards/osd/Networking/Configuration.tsx

```diff
@@ -111,7 +112,9 @@ export const Configuration = () => {
       );
 
       if (!hasFilledMachinePoolsSubnets) {
-        setFieldValue(FieldId.InstallToVpc, false);
+        if (!isOSDFromGoogleCloud) {
+          setFieldValue(FieldId.InstallToVpc, false);
+        }
         clearSecurityGroups();
```

Comment:
I'm not sure how to reproduce this scenario to test it. Do you know how to do it?